server/log.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+/**
+ * redis.js
+ *
+ * Used to store Audit logs.
+ */
+
+const Redis = require("redis");
+const logLength = 5000;
```

Comment:
agreed about LOGLENGTH being an ENV param. ok to default to 5000.

no to sending logs to Redis: logs will be stored in LogEntries (and/or Splunk)